src/main/kotlin/no/nav/syfo/application/BlockingApplicationRunner.kt

```diff
@@ -27,11 +34,43 @@ class BlockingApplicationRunner(
     val mqSender: MQSenderInterface,
     val dialogmeldingProducer: DialogmeldingProducer,
 ) {
+    val azureAdV2Client = AzureAdV2Client(
+        aadAppClient = env.aadAppClient,
+        aadAppSecret = env.aadAppSecret,
+        aadTokenEndpoint = env.aadTokenEndpoint,
+        httpClient = httpClientWithProxy,
+    )
+
+    val smgcpClient = SmgcpClient(
+        azureAdV2Client = azureAdV2Client,
+        endpointUrl = env.smgcpProxyUrl,
+        resourceId = env.smgcpProxyClientId,
+        httpClient = httpClient,
+    )
+
+    val smtssClient = SmtssClient(
+        azureAdV2Client = azureAdV2Client,
+        smtssClientId = env.smtssClientId,
+        smtssUrl = env.smtssApiUrl,
+        httpClient = httpClient,
+    )
+
+    val tssService = TssService(
+        smtssClient = smtssClient,
+    )
+
+    val emottakService = EmottakService(
+        smgcpClient = smgcpClient,
+    )
+
```

Comment:
Her initialiseres de i denne klassen, mens i `Bootstrap` ligger de i  `launchBackgroundTask` funksjonen. Er det noe problematikk rundt å ha feks to ulike instanser av AzureClienten og disse andre? Kunne de alle ha blitt initialisert i `Bootstrap` og så sendt ned som parametere?